modules/node/handler/cms_node_pagination_builder.e

```diff
@@ -0,0 +1,44 @@
+note
+	description: "Paginator builder for CMS nodes."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_NODE_PAGINATION_BUILDER
```

Comment:
I see your point, in this case,  the interface CMS_PAGINATION_BUILDER [G] is not needed anymore. 
My concern with this new approach is if it is reusable, because a new module can add his own storage strategy (new tables for example) and implement his own pagination strategy. The interface has the advantage of flexibility.